packages/ui/src/memberships/components/MemberListItem/Fileds.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Fonts, Sizes } from '../../../common/constants'
+import { Membership, OtherMember } from '../../types'
+
+export const colLayoutByType = (kind: (Membership | OtherMember)['kind']) => {
```

Comment:
```suggestion
export const colLayoutByType = (kind: 'Member' | 'MyMember') => {
```

This to be in line with the proposed changes below :point\_down: